aspnetcore/blazor/hybrid/tutorials/maui-blazor-web-app.md

```diff
@@ -16,6 +16,31 @@ This article shows you how to build a .NET MAUI Blazor Hybrid app with a Blazor
 
 For prerequisites and preliminary steps, see <xref:blazor/hybrid/tutorials/maui>. We recommend using the .NET MAUI Blazor Hybrid tutorial to set up your local system for .NET MAUI development before using the guidance in this article.
 
+:::moniker range=">= aspnetcore-9.0"
+
+## .NET MAUI Blazor Web App project template
+
+The .NET MAUI Blazor Web App project template allows you to choose a Blazor interactive render mode for the web app and creates the appropriate projects for the app, including a Blazor Web App and a .NET MAUI Blazor Hybrid app. A shared Razor class library (RCL) maintains the Razor components for the app's UI. The template also provides sample code to shows you how to use dependency injection to provide different interface implementations for the Blazor Hybrid and Blazor Web App, which is covered in the [Using interfaces to support different device implementations](#using-interfaces-to-support-different-device-implementations) section of this article.
+
+Create an app from the project template with the following .NET CLI command:
+
+```dotnetcli
+dotnet new maui-blazor-web -o MauiBlazorWeb -int Server  -ai
+```
+
+In the preceding command:
+
+* The `-o|--output` option creates a new folder for the app named `MauiBlazorWeb`.
+* The `-int|--interactivity` option sets up the interactivity location to `Server`.
+* The `-ai|--all-interactive` option specifies global interactivity, which is important because MAUI apps always run interactively and throw errors on Razor component pages that explicitly specify a render mode. If you don't use a global render mode, you must implement the approach described in the [Use Blazor render modes](#use-blazor-render-modes) section. For more information, see [BlazorWebView needs a way to enable overriding ResolveComponentForRenderMode (`dotnet/aspnetcore` #51235)](https://github.com/dotnet/aspnetcore/issues/51235).
+
+<!-- UPDATE 9.0 Provide the project template's name here for VS, possibly
+                using a tooling pivot for the article. -->
+
+:::moniker-end
+
+:::moniker range="< aspnetcore-9.0"
+
 ## .NET MAUI Blazor Web App sample app
```

Comment:
```suggestion
## .NET MAUI Blazor Hybrid and Web App sample app
```

Reply:
The only difference between the guidance below and what the solution template creates for you is that the name of the MAUI project does not have ".Maui" naming, it's just the name that the user indicated as the name of their project/solution. So if you named it "MyApp" it would be: 
  * Solution: `MyApp`
  * MAUI project: `MyApp`
  * Blazor Web App: `MyApp.Web`
  * Blazor Web Assembly App: `MyApp.Web.Client`
  * Razor class library (RCL): `MyApp.Shared`